thread/win32/omrthreadnuma.c

```diff
@@ -148,3 +148,16 @@ omrthread_numa_get_node_affinity(omrthread_t thread, uintptr_t *numaNodes, uintp
 	*nodeCount = 0;
 	return J9THREAD_NUMA_ERR_AFFINITY_NOT_SUPPORTED;
 }
+
+uintptr_t
+omrthread_numa_get_current_node(){
+    USHORT node = 0;
+
+    PROCESSOR_NUMBER processorNumber;
+	GetCurrentProcessorNumberEx(&processorNumber);
+	if (GetNumaProcessorNodeEx(&processorNumber, &node)) {
+		return node + 1;
+	}  else {
+		return 0;
```

Comment:
Should be possible to do this with a single return statement similar to the linux version